Cargo.toml

```diff
@@ -13,17 +13,17 @@ solvers = { path = "solvers" }
 game-data = { path = "game_data" }
 
 egui = "0.27.2"
-egui_extras = { version = "0.27.2", features = ["all_loaders"] }
+
 eframe = { version = "0.27.2", features = ["persistence"] }
-image = { version = "0.24.9", default-features = false, features = ["png"] }
+egui_extras = { version = "0.27.2", features = ["all_loaders"] }
+image = { version = "0.24", features = ["jpeg", "png"] }
```

Comment:
Why is `jpeg` added here? Also consider using `default-features = false` to reduce compilation time and potential binary bloat.

Reply:
Good catch. I had this when I was trying to get the skill images to load.